UPGRADING.md

```diff
@@ -0,0 +1,276 @@
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# 2.0.0 Migration Guide
+
+The 2.0 release of the `google-cloud-bigquery-storage` client is a significant
+upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python),
+and includes substantial interface changes. Existing code written for earlier versions
+of this library will likely require updates to use this version. This document
+describes the changes that have been made, and what you need to do to update your usage.
+
+If you experience issues or have questions, please file an
+[issue](https://github.com/googleapis/python-bigquery-storage/issues).
+
+
+## Supported Python Versions
+
+> **WARNING**: Breaking change
+
+The 2.0.0 release requires Python 3.6+.
+
+
+## Import Path
+
+The library was moved into `google.cloud.bigquery` namespace. It is recommended
+to use this path in order to reduce the chance of future compatibility issues
+in case the library is restuctured internally.
+
+**Before:**
+```py
+from google.cloud.bigquery_storage_v1 import BigQueryReadClient
+```
+
+**After:**
+```py
+from google.cloud.bigquery.storage import BigQueryReadClient
+```
+
+
+## Enum Types
+
+> **WARNING**: Breaking change
+
+Enum types have been moved. Access them through the `types` module.
+
+**Before:**
+```py
+from google.cloud.bigquery_storage_v1 import enums
+
+data_format = enums.DataFormat.ARROW
+```
+
+data_format = BigQueryReadClient.enums.DataFormat.ARROW
+
+**After:**
+```py
+from google.cloud.bigquery.storage import types
+
+data_format = types.DataFormat.ARROW
+```
+
+Additionally, enums cannot be accessed through the client anymore. The following
+code wil _not_ work:
+```py
+data_format = BigQueryReadClient.enums.DataFormat.ARROW
+```
+
+
+## Clients for Beta APIs
+
+> **WARNING**: Breaking change
+
+Clients for beta APIs have been removed. The following import will _not_ work:
+
+```py
+from google.cloud.bigquery_storage_v1beta1 import BigQueryStorageClient
+from google.cloud.bigquery_storage_v1beta2.gapic.big_query_read_client import BigQueryReadClient
+```
+
+The beta APIs are still available on the server side, but you will need to use
+the 1.x version of the library to access them.
+
+
+## Changed Default Value of the `read_rows()` Method's `metadata` Argument
+
+The `client.read_rows()` method does not accept `None` anymore as a valid value
+for the optional `metadata` argument. If not given, an empty tuple is used, but
+if you want to explicitly pass an "empty" value, you should use an empty tuple, too.
+
+**Before:**
+```py
+client.read_rows("stream_name", metadata=None)
+```
+
+**After:**
+```py
+client.read_rows("stream_name", metadata=())
+```
+
```

Comment:
Let's emphasize that the default value is still valid.

```suggestion

OR

```py
client.read_rows("stream_name")
```
```